### PR TITLE
Default version update rn

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/40020

## Description
Same release notes template for all content items

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,7 +42,6 @@ the same as the last release note version.
 * Fixed an issue in the **validate** and **format** commands where *None* arguments in integration commands caused the commands to fail unexpectedly.
 * Added support for running tests on XSIAM machines in the **test-content** command.
 * Fixed an issue where the **validate** command did not work properly when deleting non-content items.
-* Added the flag '-d', '--dependency' to **find-dependencies** command to get the content items that cause the dependencies between two packs.
```
Comment (dorschw) [inline CHANGELOG.md:45]: seems unintentional, please revert

File: demisto_sdk/commands/common/tools.py [hunk 2]
```diff
@@ -606,18 +606,20 @@ def get_from_version(file_path):
...
     if data_dictionary:
+
         from_version = data_dictionary.get('fromversion') if 'fromversion' in data_dictionary \
-            else data_dictionary.get('fromVersion', DEFAULT_CONTENT_ITEM_FROM_VERSION)
-        if from_version == '':
-            return DEFAULT_CONTENT_ITEM_FROM_VERSION
+            else data_dictionary.get('fromVersion', '')
+
+        if not from_version:
+            return from_version
```
Comment (yuvalbenshalom) [inline demisto_sdk/commands/common/tools.py:614]:
```suggestion
        if not from_version:
            return ''
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comment

File: demisto_sdk/commands/common/tools.py (hunk 2 above)
Comment (rshunim) [reply demisto_sdk/commands/common/tools.py]: done

File: demisto_sdk/commands/common/tools.py [hunk 3]
```diff
@@ -606,18 +606,20 @@ def get_from_version(file_path):
...
+            return ''
 
         if not re.match(r'^\d{1,2}\.\d{1,2}\.\d{1,2}$', from_version):
             raise ValueError(f'{file_path} fromversion is invalid "{from_version}". '
                              'Should be of format: "x.x.x". for example: "4.5.0"')
 
         return from_version
 
-    return DEFAULT_CONTENT_ITEM_FROM_VERSION
+    return ''
```
Comment (dorschw) [inline demisto_sdk/commands/common/tools.py:622]: please add `logging.warning` or `logging.debug` (choose based on how much it spams the log) that the `fromversion` was not found, adding the content item name.

Comment (rshunim) [reply demisto_sdk/commands/common/tools.py:623]: done

File: demisto_sdk/commands/common/tools.py [hunk 4]
```diff
@@ -606,18 +606,21 @@ def get_from_version(file_path):
     data_dictionary = get_yaml(file_path) if file_path.endswith('yml') else get_json(file_path)
 
     if data_dictionary:
+
```
Comment (DeanArbel) [inline demisto_sdk/commands/common/tools.py:609]: why is there space here?

**Review by DeanArbel — APPROVED** [review]
Looks good, just one small change.

**Pull request merged**